doc/classes/Basis.xml

```diff
@@ -159,13 +173,13 @@
 		<method name="inverse" qualifiers="const">
 			<return type="Basis" />
 			<description>
-				Returns the inverse of the matrix.
+				Returns the [url=https://en.wikipedia.org/wiki/Invertible_matrix]inverse of this basis' matrix[/url].
 			</description>
 		</method>
 		<method name="is_conformal" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the basis is conformal, meaning it preserves angles and distance ratios, and may only be composed of rotation and uniform scale. Returns [code]false[/code] if the basis has non-uniform scale or shear/skew. This can be used to validate if the basis is non-distorted, which is important for physics and other use cases.
+				Returns [code]true[/code] if this basis is conformal. A conformal basis only represents rotation or scale, with no shearing, and thus it is not distorted. This method can be especially useful during physics calculations.
```

Comment:
This is not correct. Conformal requires uniform scale. If the scale is not uniform, it is not conformal.